### PR TITLE
Add DDD implementation article

### PR DESCRIPTION
**Pull request by simara-svatopluk**
I'm not sure about links in the article. In-text links the pehapkari GitHub, but links in the last section link to my GitHub. What do You think?

And I considered including code into the article, but it would look so bad, so I decided to just link the code.

@TomasVotruba May I ask You to check this article?

### CODE REVIEW

File: easy-coding-standard.neon [hunk 1]
```diff
@@ -37,6 +31,7 @@ parameters:
         # from symplify-checkers.neon set
         - Symplify\CodingStandard\Fixer\Naming\PropertyNameMatchingTypeFixer
         - Symplify\CodingStandard\Sniffs\Property\DynamicPropertySniff
+        - Symplify\CodingStandard\Sniffs\DependencyInjection\NoClassInstantiationSniff
```
Comment (TomasVotruba) [inline easy-coding-standard.neon]: :+1:

**Comment by TomasVotruba** [comment]
Thank you

**Comment by TomasVotruba** [comment]
> In-text links the pehapkari GitHub, but links in the last section link to my GitHub. What do You think?

If you prefer that, then it's ok. Personally I try to keep code and text in one place, because I cleanup old repositories and might forget it's linked in another repository.

I've redated to week after last added post in queue.

Is it ready to merge?

**Comment by simara-svatopluk** [comment]
yes, thanks :-)

**Comment by TomasVotruba** [comment]
Thank you for your series. Your probably holding a record :+1:

**Comment by simara-svatopluk** [comment]
Thanks for support - mental and also review/merge :-)

**Pull request merged**